packages/react/src/components/Button/Button.stories.mdx

```diff
@@ -194,6 +211,52 @@ On through of which no the your phase more know anyone tried he when line me wha
     </Story>
 </Canvas>
 
+### Bruk av Spinner til å lage en knapp som laster
```

Comment:
Good story and very good description! 

I would also have liked an example with a simple timeout to simulate a backend call so we can also test with screen-readers when it "re-activates" but mdx is so horrible to work in terms of writing code which I wouldn't inflict on anyone 🙈 Storybook v7 is out soon so we can save it for when we get this updated :)

Reply:
Haha thanks! 🙈